file.c

```diff
@@ -53,6 +53,18 @@ void openFile(void)
       fileSize = 0;
   }
   biggestLoc = fileSize;
+
+  // compute number of digits for address
```

Comment:
I suggest adding a comment somewhere explaining that nAddrDigits max is 16 (max size is `2**64` and a hex char is `2**4` wide)

Reply:
Could you move this code to a function `compute_nAddrDigits` (or something alike)